pom.xml

```diff
@@ -9,15 +9,14 @@
     <description>This projects contains the Exasol dialect for Exasol's Virtual Schema</description>
     <url>https://github.com/exasol/exasol-virtual-schema</url>
     <properties>
-        <product.version>3.0.0</product.version>
+        <product.version>3.0.1</product.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
         <junit.version>5.6.1</junit.version>
         <junit.platform.version>1.6.1</junit.platform.version>
         <surefire.and.failsafe.plugin.version>3.0.0-M4</surefire.and.failsafe.plugin.version>
-        <vscjava.version>10.0.0</vscjava.version>
-        <vscjdbc.version>5.0.0</vscjdbc.version>
+        <vscjdbc.version>5.0.1</vscjdbc.version>
```

Comment:
:+1: 
Kudos, you even remembered to remove the `vscjava` dependency!

Reply:
But please remove the direct dependency from the README then too. It's now a transient dependency.